cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/etl/ScriptConstants.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package co.cask.cdap.etl;
+
+/**
+ * Constants for transforms using JavaScript.
+ */
+public class ScriptConstants {
+  public static final String HELPER_NAME = "dont_name_your_helper_this";
```

Comment:
Better be something like "CDAP_ETL_SCRIPT_LIB". It's all cap because it's a global object in the script env, right?